salt/cloud/clouds/libvirt.py

```diff
@@ -462,18 +462,39 @@ def create(vm_):
 
         return ret
     except Exception as e:  # pylint: disable=broad-except
-        # Try to clean up in as much cases as possible
-        log.info('Cleaning up after exception clean up items: {0}'.format(cleanup))
-        for leftover in cleanup:
-            what = leftover['what']
-            item = leftover['item']
-            if what == 'domain':
-                destroy_domain(conn, item)
-            if what == 'volume':
-                item.delete()
+        do_cleanup(cleanup)
         raise e
 
 
+def do_cleanup(cleanup):
+    # Try to clean up in as many cases as possible. Libvirt behavior changed
```

Comment:
A docstring for this function would be good. Can you also make sure the version added tag is included here?
```
... versionadded:: 2017.7.3
```

Reply:
I believe it is actually `.. versionadded: 2017.7.3`

With just 2 dots at the start